libs/usb-drive/src/mocks/memory_usb_drive.ts

```diff
@@ -74,6 +75,7 @@ export function createMockUsbDrive(): MockUsbDrive {
     removeUsbDrive() {
       mockUsbTmpDir?.removeCallback();
       mockUsbTmpDir = undefined;
+      usbDrive.status.reset();
```

Comment:
Without these resets, inserting and removing the mock USB drive continues growing the list of expected repeated calls. This results in unexpected behavior like the mock USB drive indicating that it's no drive even after calling `mockUsbDrive.insertUsbDrive`